include/socgraph.php

```diff
@@ -1244,18 +1244,20 @@ function poco_discover_federation() {
 			poco_check_server("https://".$server->host);
 	}
 
-	// Discover GNU Social Servers
```

Comment:
Maybe there is an alternative way, ask mmn in the fediverse, if there is a way to discover all known instances.